towerlib/entities/organization.py

```diff
@@ -512,6 +513,56 @@ def create_inventory(self, name, description, variables='{}'):
             self._logger.error('Error creating inventory "%s", response was "%s"', name, response.text)
         return Inventory(self._tower, response.json()) if response.ok else None
 
+    @property
+    def inventory_scripts(self):
+        """The inventory scripts of the organization.
+
+        Returns:
+            EntityManager: EntityManager of the inventory scripts of the organization.
+
+        """
+        url = '{organization}inventory_scripts/'.format(organization=self.api_url)
+        return EntityManager(self._tower,
+                             entity_object='inventory_scripts',
+                             primary_match_field='name',
+                             url=url)
+
+    def get_inventory_script_by_name(self, name):
+        """Retrieves an custom inventory script.
+
+        Args:
+            name: The name of the custom inventory script to retrieve.
+
+        Returns:
+            inventory(Inventory): custom inventory script on success else None.
+
+        """
+        return next(self._tower.inventories_scripts.filter({'organization': self.id, 'name__iexact': name}), None)
+
+    def create_inventory_script(self, name, description, script):
+        """Creates a custom inventory script.
+
+        Args:
+            name: Name of the inventory script.
+            description: The description of the inventory script.
+            script: The script of the inventory script.
+
+        Returns:
+            Inventory_script: The created inventory script is successful, None otherwise.
+
+        """
+        url = '{api}/inventory_scripts/'.format(api=self._tower.api)
+        payload = {'name': name,
+                   'description': description,
+                   'inventory': self.id,
```

Comment:
This is probably not correct, i don't think that the inventory id and the organization id should be the same value.

Reply:
You are quite right, inventory should not be part of the body. Will remove and add it to the pull request